build/angular-slider.js

```diff
@@ -943,10 +954,10 @@ angular.module('vr.directives.slider', ['ngTouch']).directive('slider',
 
                                 // save the various dimensions we'll need
                                 barWidth = width(refs.fullBar);
-                                pointerHalfWidth = halfWidth(refs.minPtr);
+                                pointerHalpointerHalfWidth = parseInt(minPtrWidth / 2);
```

Comment:
Personally I think `minPtrWidth` should be renamed to `minPointerWidth` so it can be related to `pointerHalfWidth`.